bedrock/newsletter/views.py

```diff
@@ -303,10 +324,13 @@ def existing(request, token=None):
     #  u'email': u'user@example.com'
     # }
 
-    user_exists = False
+    has_fxa = 'fxa' in request.GET
```

Comment:
Is there any concern if someone is not an FxA holder but somehow has `fxa` in the URL? Could that result in a bug if they then try to view/subscribe to FxA newsletters?

Reply:
I don't think so. This is really just a measure to only promote a couple of newsletters to people who have a FxA. If they somehow get the FxA link and see them then my understanding is that it's no big deal. They aren't secret, they just are only promoted as part of an FxA. The query param is really an efficiency thing so that basket doesn't have to look up the FxA in the case when we know they have one, and this param should only be set when coming from an FxA email or within the FxA UI itself.